sdk/python/kfp/_client.py

```diff
@@ -277,9 +285,22 @@ def set_user_namespace(self, namespace):
       namespace: kubernetes namespace the user has access to.
     """
     self._context_setting['namespace'] = namespace
+    if not os.path.exists(os.path.dirname(Client.LOCAL_KFP_CONTEXT)):
+        os.makedirs(os.path.dirname(Client.LOCAL_KFP_CONTEXT))
     with open(Client.LOCAL_KFP_CONTEXT, 'w') as f:
       json.dump(self._context_setting, f)
 
+  def get_kfp_healthz(self):
+    """Get healthz info of KFP deployment.
+
+    Returns:
+      response: json formatted response from the healtz endpoint.
+    """
+    healthz_api = 'http://' + Client.IN_CLUSTER_DNS_NAME.format('kubeflow') + '/' + Client.HEALTH_PATH
+    r = requests.get(healthz_api)
```

Comment:
shall we add a `r.raise_for_status()` to abort on error?
https://requests.readthedocs.io/en/master/user/quickstart/#errors-and-exceptions

Reply:
Added